### ISSUE TITLE
Dash to Dock causing Gnome Shell to crash upon locking screen

### ISSUE THREAD

**Issue by coralof** [pop-os/shell#915]
**Distribution (run `cat /etc/os-release`):**
 
NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os

 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
Dash to Dock Gnome Shell extension

**Issue/Bug Description:**
Whenever this extension is enabled, it causes Gnome Shell to completely crash whenever the screen is locked. Forces a hard shutdown, and requires the computer to be restarted. Upon disabling and uninstalling the extension, the issue is fixed. Already reported on Dash to Dock's extension page as well. Unsure if it's an issue with specifically the extension itself, or a combination of the extension and Pop_OS' Gnome shell.

**Steps to reproduce (if you know):**
Enable Dash to Dock extension (GNOME 3.36.8, Pop_OS 20.04), and lock the screen.

**Expected behavior:**
Normal locking behavior; lock screen keeps computer awake, and when a key is pressed, lock screen comes back up.

**Other Notes:**

**Comment by coralof** [comment]
Hello,

For the dash-to-dock settings, I had the following selected:

Position and Size

- Position on screen: Left
- Intelligent Autohide: Off
- Dock size limit: extend to the screen edge
- Icon Size: 32
- Fixed icon size: scroll to reveal other icons is SELECTED

Launchers

- Show favorite applications ON
- Show running applications ON
- Show open windows previews CHECKED
- Show Applications icon ON
- Animate Show Applications CHECKED

Behavior

- Use keyboard shortcuts to active Apps OFF
- Click Action: Minimize or overview
- Scroll action: Do nothing

Appearance

- Use built-in theme OFF
- Shrink the Dash ON
- Customize Dash color ON (black)
- Force straight corner OFF

I was using Pop_OS 20.04 when this happened. Let me know if this helps.

Thanks,
Cora

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by bflanagin** [comment]
Hmmm, not seeing the issue here on 20.04 (yet) I'll keep at it to make sure there isn't something. On your end, try completely uninstalling the extension and reinstalling it. It may not help, but I want o make sure we're on the same version of Dash-to-dock.

**Comment by coralof** [comment]
Hi,

I ended up rolling my install back with timeshift to just after I finished installing (since it was a fresh install anyway), I reinstalled a fresh copy of Dash to Dock, set all the same settings as before, locked my screen a couple times, rebooted and locked the screen, had multiple applications running, etc. It seems to be working fine now! I have no idea what it could have been.

Thanks for looking into it. I do see other threads, like [this one](https://github.com/micheleg/dash-to-dock/issues/812) that report similar issues with Dash to Dock, so I’m thinking it’s probably not Pop-Shell that’s causing the issue.

Regards,
Cora

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
